tests/rptest/tests/mirror_maker_test.py

```diff
@@ -8,38 +8,60 @@
 # by the Apache License, Version 2.0
 import random
 from ducktape.utils.util import wait_until
-from ducktape.mark import ignore
-from ducktape.mark.resource import cluster
+from rptest.services.cluster import cluster
+from ducktape.mark import parametrize
 
-import time
 from rptest.clients.default import DefaultClient
-from rptest.clients.rpk import RpkTool
 from rptest.clients.types import TopicSpec
-from rptest.clients.kcl import KCL
+from rptest.clients.rpk import RpkTool
+from rptest.services.rpk_consumer import RpkConsumer
+from rptest.services.rpk_producer import RpkProducer
+from rptest.services.kafka import KafkaServiceAdapter
 from rptest.services.mirror_maker2 import MirrorMaker2
 
 from rptest.services.redpanda import RedpandaService
-from rptest.services.rpk_consumer import RpkConsumer
-from rptest.services.rpk_producer import RpkProducer
 from rptest.tests.end_to_end import EndToEndTest
 from rptest.services.verifiable_producer import VerifiableProducer, is_int_with_prefix
 from rptest.services.verifiable_consumer import VerifiableConsumer
+from kafkatest.services.kafka import KafkaService
+from kafkatest.services.zookeeper import ZookeeperService
+
+from kafkatest.version import V_3_0_0
 
 
 class TestMirrorMakerService(EndToEndTest):
+    kafka_source = "kafka"
+    redpanda_source = "redpanda"
```

Comment:
nit: python has an enum thingy

Reply:
python enum type is not json serializable :(

Reply:
it has to be to dump json tests results as we use it as parameter